tests/base/time_stepping_01.cc

```diff
@@ -266,6 +266,32 @@ int
 main()
 {
   initlog();
+  deallog.precision(4);
+  {
+    const std::vector<double> bi = {{1153189308089. / 22510343858157.,
+                                     1772645290293. / 4653164025191.,
+                                     -1672844663538. / 4480602732383.,
+                                     2114624349019. / 3568978502595.,
+                                     5198255086312. / 14908931495163.}};
+    std::vector<double>       ai;
+    ai = {{970286171893. / 4311952581923.,
+           6584761158862. / 12103376702013.,
+           2251764453980. / 15575788980749.,
+           26877169314380. / 34165994151039.}};
+
+    deallog << "Check low-storage Runge-Kutta coefficients" << std::endl;
+    TimeStepping::LowStorageRungeKutta<Vector<double>> lsrk54(
+      TimeStepping::LOW_STORAGE_RK_STAGE5_ORDER4);
+    std::vector<double> a, b, c;
+    lsrk54.get_coefficients(a, b, c);
+
+    for (unsigned int i = 0; i < b.size(); ++i)
+      {
+        deallog << " b: " << b[i] - bi[i] << std::endl;
+        if (i != b.size() - 1)
+          deallog << " a: " << a[i] - ai[i] << std::endl;
```

Comment:
you could print ``c`` here, at least the values will be checked in the test then.